files/fluentd/run.sh

```diff
@@ -4,26 +4,8 @@ export MERGE_JSON_LOG=${MERGE_JSON_LOG:-false}
 CFG_DIR=/etc/fluent/configs.d
 ENABLE_PROMETHEUS_ENDPOINT=${ENABLE_PROMETHEUS_ENDPOINT:-"true"}
 OCP_OPERATIONS_PROJECTS=${OCP_OPERATIONS_PROJECTS:-"default openshift openshift- kube-"}
-LOGGING_FILE_PATH=${LOGGING_FILE_PATH:-"/var/log/fluentd/fluentd.log"}
-
-loggingargs=""
-if [ ${LOGGING_FILE_PATH} != "console" ] ; then
-    echo "============================="
-    echo "Fluentd logs have been redirected to: $LOGGING_FILE_PATH"
-    echo "If you want to print out the logs, use command:"
-    echo "oc exec <pod_name> -- logs"
-    echo "============================="
-
-    dirname=$( dirname $LOGGING_FILE_PATH )
-    if [ ! -d $dirname ] ; then
-        mkdir -p $dirname
-    fi
-    touch $LOGGING_FILE_PATH; exec >> $LOGGING_FILE_PATH 2>&1
-
-    loggingargs="-o $LOGGING_FILE_PATH --log-rotate-age $LOGGING_FILE_AGE --log-rotate-size $LOGGING_FILE_SIZE"
-fi
 
-fluentdargs="--no-supervisor $loggingargs"
+fluentdargs="--no-supervisor"
```

Comment:
so i was actually just thinking about this... and this is probably a larger discussion but if we're logging to stdout how long does the container hold this information? and what happens to our running pod size the longer it lives? do we expect to just roll them periodically? or can we configure it to age out older stdout entries?

Reply:
> so i was actually just thinking about this... and this is probably a larger discussion and not a blocker but if we're logging to stdout how long does the container hold this information? and what happens to our running pod size the longer it lives? do we expect to just roll them periodically? or can we configure it to age out older stdout entries?

I believe this is the responsibility of the runtime to manage the logfile and its rotation